RIM/model_interface.py

```
@@ -0,0 +1,49 @@
+            print(f"Response object: {response}")
+            print(f"Choices: {response.choices}")
+
+            content = response.choices[0].message.content
+            print(f"Content: {content}")
+            print(f"Content type: {type(content)}")
+
+            if content is None:
+                print("WARNING: Content is None")
+                return json.dumps({"score_a": 0.5, "score_b": 0.5, "explanation": "Empty response", "uncertainty": 1.0})
+
+            return content
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)